contract_manager/src/contracts/evm.ts

```diff
@@ -622,6 +633,52 @@ export class EvmEntropyContract extends Storable {
     };
   }
 
+  async getRequest(
```

Comment:
nit: a 1 sentence doc comment on these methods would be nice. (esp for getUserRandomNumber, would be nice to explain how to get the `block` argument if you don't have it)